docs/queries/about.mdx

```diff
@@ -110,13 +110,14 @@ possible. With this in mind, we recommend this order of priority:
       `getByRole('button', {name: /submit/i})`. Check the
       [list of roles](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques#Roles).
    1. `getByLabelText`: Only really good for form fields, but this is the number
-      one method a user finds those elements, so it should be your top
-      preference.
+      one method to make sure that a user finds those elements, so it should be
```

Comment:
it's more like "users use labels to find form fields; this method emulates that"

Reply:
Ah, okay!

I'll revert the changes. Are other minor changes legit?

Reply:
The change to getByText has the same issue.

Reading through the text, though, all of the phrasing could use a little work on the grammar. Open to updates.